client/notices.go

```diff
@@ -181,6 +193,13 @@ func makeNoticesQuery(opts *NoticesOptions) url.Values {
 	if opts == nil {
 		return query
 	}
+	if len(opts.UserIDs) > 0 {
+		userIDStrs := make([]string, 0, len(opts.UserIDs))
+		for _, u := range opts.UserIDs {
+			userIDStrs = append(userIDStrs, strconv.FormatInt(int64(u), 10))
```

Comment:
This is equivalent to `Itoa`:

```suggestion
			userIDStrs = append(userIDStrs, strconv.Itoa(u))
```